### PR TITLE
Connects Falkonry/tracker#814

### PR DESCRIPTION
**Pull request by dilpeshjain**

### CODE REVIEW

File: src/main/java/com/falkonry/client/service/HttpService.java [hunk 1]
```diff
@@ -40,15 +40,15 @@
 	public HttpService(String host, String token) throws Exception {
 
 		// for localhost testing only
-		/*javax.net.ssl.HttpsURLConnection.setDefaultHostnameVerifier(new javax.net.ssl.HostnameVerifier() {
```
Comment (aniket-amrutkar) [inline src/main/java/com/falkonry/client/service/HttpService.java]: add comment

File: src/test/java/com/falkonry/TestAddAndGetData.java [hunk 2]
```diff
@@ -108,12 +222,13 @@ public void addDataJson() throws Exception {
...
 	    options.put("responseFormat", "text/csv");  // also available options 1. text/csv 2. application/json
-		HttpResponseFormat dataResponse = falkonry.getInputData("qh6rg4ce5g3p5j", options);
+		HttpResponseFormat dataResponse = falkonry.getInputData(datastream.getId(), options);
 		Assert.assertEquals(dataResponse.getResponse().length()>0,true);
 	}
 
+	
 	/**
-	 * Should add data to datastream in CSV format
+	 * Add wide input data (json format) to single thing Datastream
```
Comment (aniket-amrutkar) [inline src/test/java/com/falkonry/TestAddAndGetData.java]: replace thing with entity in all comments, readme

File: src/test/java/com/falkonry/TestAddAndGetFacts.java [hunk 3]
```diff
@@ -219,10 +223,17 @@ public void createDatastreamWithAdditionalTagsCsvFacts() throws Exception {
...
 
         Map<String, String> options = new HashMap<String, String>();
-        String data = "{\"time\" : \"2016-03-01T01:01:01Z\", \"tag\" : \"signal1_entity1\", \"value\" : 3.4}";
+        String data = "{\"time\" : \"2016-03-01T01:01:01Z\", \"signal\" : \"signal1\", , \"entity\" : \"entity1\", \"value\" : 3.4}";
         options.put("timeIdentifier", "time");
         options.put("timeFormat", "iso_8601");
-        options.put("fileFormat", "csv");
+        options.put("timeZone", time.getZone());
+        options.put("fileFormat", "json");
+		options.put("signalIdentifier", "signal");
```
Comment (aniket-amrutkar) [inline src/test/java/com/falkonry/TestAddAndGetFacts.java]: ident properly in all the files

File: src/main/java/com/falkonry/helper/models/StatsMeasurementMap.java [hunk 4]
```diff
@@ -62,36 +62,36 @@ public void setSignal(String signal) {
...
     /**
      *
-     * @param thing
+     * @param entity
      */
-    @JsonProperty("Thing")
-  public void setThing(String thing) {
-    this.Thing = thing;
+    @JsonProperty("entity")
+  public void setentity(String entity) {
```
Comment (phagunbaya) [inline src/main/java/com/falkonry/helper/models/StatsMeasurementMap.java]: Typo `setEntity`

**Comment by phagunbaya** [comment]
Please fix the camel casing.

**Pull request merged**